src/client/src/serviceWorker.js

```diff
@@ -30,8 +30,7 @@ export function register(config) {
     }
 
     window.addEventListener('load', () => {
-      const swUrl = `${process.env.PUBLIC_URL}/service-worker.js`
-
+      const swUrl = `${process.env.PUBLIC_URL}/rtc-sw.js`
```

Comment:
What is the difference bewteen service-workes.js and rtc-sw.js @pmilla1606? Do we need both?

Reply:
`serviceWorker.js` is the default created by `create-react-app` which is in charge of managing service workers. `rtc-sw.js` is a minimal service worker that will allow us to meet the installability requirements. I think it's a good idea to keep them separate especially if we extend this in the future.